public/caesar-shift/decrypt.py

```diff
@@ -0,0 +1,32 @@
+import string
+from encrypt_library import *
+from encrypt import caesar_shift
+#START#
+def caesar_shift_decrypt(ciphertext, shift_value=5):
+  #we can flip the shift value we think the encryptor used to decrypt the message!
+  shift_value = -shift_value
```

Comment:
I'd recommend tightening it up by just making this a single-line function:
```
return caesar_shift_decrypt(ciphertext, -shift_value)
```

I think it's tighter & about as transparent about what it's doing as shift_value = -shift_value; if they understand one, they understand the other (I would argue). Thoughts? I'm open to pushback, btw.